MAUI/Scheduler/migration.md

```diff
@@ -878,6 +946,79 @@ this.Content = scheduler;
 <td>Gets or sets a view header day format of the SfScheduler.</td></tr>
 
  </table> 
+ 
+The following code example, explains how to initialize the properties of the Xamarin SfSchedule and .NET MAUI SfScheduler class.
```

Comment:
The following code example explains how to configure the days view settings in Xamarin SfSchedule and .NET MAUI SfScheduler.